### PR TITLE
Add constant-time 64

### PR DESCRIPTION
**Pull request by richsalz**
This replaces GitLab MR 1236 from @ekasper

### CODE REVIEW

**Comment by richsalz** [comment]
Fixed and taken out of WIP.  Ready for review.

File: include/internal/constant_time_locl.h [hunk 1]
```diff
@@ -41,6 +41,7 @@ extern "C" {
  * replace this with something else on odd CPUs.
  */
 static ossl_inline unsigned int constant_time_msb(unsigned int a);
+static ossl_inline uint64_t constant_time_msg_64(uint64_t a);
```
Comment (dot-asm) [inline include/internal/constant_time_locl.h]: msg?

**Comment by richsalz** [comment]
Hah, at first I thought you were being terse and asking for a message – that is, a comment.  Then I saw that I had a typo ☺

Additional commit pushed which fixes up the comments to be similar structure.  And fixes the typo.

**Comment by richsalz** [comment]
thanks for the review!

**Pull request closed without merging**